pwiz_tools/Skyline/Model/Irt/RCalcIrt.cs

```diff
@@ -144,12 +144,17 @@ public override IEnumerable<Target> ChooseRegressionPeptides(IEnumerable<Target>
         {
             RequireUsable();
 
-            var returnStandard = peptides.Where(_database.IsStandard).Distinct().ToArray();
+            var pepArr = peptides.ToArray();
+            var returnStandard = pepArr.Where(_database.IsStandard).Distinct().ToArray();
             var returnCount = returnStandard.Length;
             var databaseCount = _database.StandardPeptideCount;
 
             if (!IsAcceptableStandardCount(databaseCount, returnCount))
+            {
+                Console.Out.WriteLine(@"Database standards: {0}", string.Join(@"; ", _database.StandardPeptides.Select(pep => pep.ToString())));

```

Comment:
The pep.ToString() is not necessary. You can just do:
string.Join(@"; ", _database.StandardPeptides)